# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1465340072 25200
#      Tue Jun 07 15:54:32 2016 -0700
# Branch stable
# Node ID 12b31cb1d6b71da61d8ee44e5e0de50a286cda76
# Parent  e56ba70108d37f8977fc9081b8b2deb1e91bc21b
Update gnulib subrepo for texinfo formatting fixes (bug #48001)

* .hgsubstate: Update gnulib subrepo for latest texinfo.tex macros.
* load-save.cc (write_header): Adapt to gnulib change to nstrftime arguments.

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-7f19e7f2afa2b1476983003f9cd6df2f5b2a4b08 gnulib-hg
+c2b547926f344f971af228b730451570bc752780 gnulib-hg
diff --git a/libinterp/corefcn/load-save.cc b/libinterp/corefcn/load-save.cc
--- a/libinterp/corefcn/load-save.cc
+++ b/libinterp/corefcn/load-save.cc
@@ -1207,17 +1207,17 @@ write_header (std::ostream& os, load_sav
         time_t now;
         char headertext[128];
 
         time (&now);
         bdt = *gnulib::gmtime (&now);
         memset (headertext, ' ', 124);
         // ISO 8601 format date
         nstrftime (headertext, 124, "MATLAB 5.0 MAT-file, written by Octave "
-                   OCTAVE_VERSION ", %Y-%m-%d %T UTC", &bdt, 1, 0);
+                   OCTAVE_VERSION ", %Y-%m-%d %T UTC", &bdt, 0, 0);
 
         // The first pair of bytes give the version of the MAT file
         // format.  The second pair of bytes form a magic number which
         // signals a MAT file.  MAT file data are always written in
         // native byte order.  The order of the bytes in the second
         // pair indicates whether the file was written by a big- or
         // little-endian machine.  However, the version number is
         // written in the *opposite* byte order from everything else!
